CMakeLists.txt

```diff
@@ -43,16 +43,28 @@ IF (NOT GIT_DESCRIBE_VERSION)
         OUTPUT_STRIP_TRAILING_WHITESPACE)
 ENDIF()
 
+IF (NOT GIT_SHA)
```

Comment:
why not use existind gitversion?

Reply:
Because it contains the 'git describe' output, I do not want QA to start parsing it and neither I want to parse it. I think its better this way.